internal/config/config.go

```diff
@@ -85,6 +85,7 @@ const (
 	PENDO_REQUEST_TIMEOUT                        = "Pendo_Request_Timeout"
 	PENDO_INTEGRATION_KEY                        = "Pendo_Integration_Key"
 	PENDO_REQUEST_SIZE                           = "Pendo_Request_Size"
+	PROMETHEUS_PUSHGATEWAY                       = "Prometheus_PushGateway"
```

Comment:
Nit-picky (i'm sorry!!) - can you add an underscore between push and gateway?  Please  :)